### PR TITLE
Real Fix #4 🤨

### LINKED ISSUE #4: Browser opening leaves url bar empty if the internal url is invalid
If browser was opened internaly on invalid url, the url bar becomes empty.
It can be reproduced by changing the default page to invalid url.

I found it while i was creating my pull request #3 in which i added ability to open browser with non-default page by command.
Firstly i thought it was bug that i've created, but then i've tested it the way i mentioned earlyer and got sure i had not.

### PR DESCRIPTION
**Pull request by HiWord9**
The problem on calling BrowserScreen, not on which request u give to it

### CODE REVIEW

**Comment by HiWord9** [comment]
also added support for instant download resource packs on modrinth.com/resourcepack same as modrinth.com/mod in [c332fa6](https://github.com/Blobanium/MCBrowser/pull/5/commits/c332fa6f4bf3528c2ac0edbe26abedd4fccc0e77)

File: src/main/resources/assets/mcbrowser/lang/en_us.json
Comment (Blobanium) [inline src/main/resources/assets/mcbrowser/lang/en_us.json:1]: Try not to duplicate translations. I can see some entries on lines 12, 14, 16 and 17 that can be reused on 6, 8, 10 and 11 respectfully. This is so translations from previous versions can still be good in future versions (if you can understand what i mean im not good at english xd). It also cuts down on mod size by a marginal amount as well.

**Review by Blobanium — CHANGES_REQUESTED** [review]

**Review by Blobanium — APPROVED** [review]
Changes look good! Here's your stamp of approval.

**Comment by Blobanium** [comment]
Anything else you might want to add before i merge this PR by any chance?

**Comment by HiWord9** [comment]
I I'd like to add something else I will make new PR lol

**Pull request merged**